docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
       - NET_ADMIN # For Fail2Ban to work
     restart: always
 ```
-
-## A Detailed Example
-
-!!! note
-
-    This is a community contributed guide. Please let us know via a Github Issue if you're having any difficulty following the guide so that we can update it.
-
-This guide is focused on only using [SMTP ports (not POP3 and IMAP)][docs-ports] with the intent to send received mail to another MTA service such as _Gmail_. It is not intended to have a MUA client (_eg: Thunderbird_) to retrieve mail directly from `docker-mailserver` via POP3/IMAP.
-
-In this setup `docker-mailserver` is not intended to receive email externally, so no anti-spam or anti-virus software is needed, making the service lighter to run.
-
-!!! warning "Open Relays"
-
-    Adding the docker network's gateway to the list of trusted hosts (_eg: using the `network` or `connected-networks` option_), can create an [**open relay**](https://en.wikipedia.org/wiki/Open_mail_relay). For instance [if IPv6 is enabled on the host machine, but not in Docker][github-issue-1405-comment].
-
-1. If you're running a version of `docker-mailserver` earlier than v10.2, [you'll need to get `setup.sh`][docs-setup-script]. Otherwise you can substitute `./setup.sh <command>` with `docker exec mailserver setup <command>`.
-
-2. Pull the docker image: `docker pull docker.io/mailserver/docker-mailserver:latest`.
-
-3. Create the file `docker-compose.yml` with a content like this:
-
-    !!! example
-
-        ```yaml
-        services:
-          mailserver:
-            image: docker.io/mailserver/docker-mailserver:latest
-            container_name: mailserver
-            # Provide the FQDN of your mail server here (Your DNS MX record should point to this value)
-            hostname: mail.example.com
```

Comment:
It's common to see bug reports with a misunderstanding of `hostname` + `domainname` fields in `docker-compose.yaml`. We haven't yet collapsed the two from the example file like above (probably should).

Reply:
Added to my TODO list.